app/Http/Controllers/Store/RegistrationController.php

```diff
@@ -195,6 +195,18 @@ public function edit(Registration $registration)
         } else {
             abort(404, 'Registration not found.');
         }
+        $evaluations = $registration->centre->sponsor->evaluations;
+        $is_scottish = false;
+        foreach ($evaluations as $key => $evaluation) {
+          if ($evaluation['name'] == 'ScottishChildCanDefer') {
+            $is_scottish = true;
+          }
```

Comment:
we can put `break;` here to exit a foreach loop early, when we find what we want.